packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagebox/_index.scss

```diff
@@ -0,0 +1,9 @@
+.dx-chat-message-box {
+  display: flex;
```

Comment:
```suggestion
  display: inline-flex;
```

Reply:
For what? I think it should occupy the entire available width of the container horizontally, i.e. it should behave like a block element

Reply:
Actually, it's doesn't matter in this case. I only wanted to see, that for this container I expects that all child will in one row. Let's skip this change